styles.css

```diff
@@ -162,3 +155,21 @@ body.dark-mode .expansion p {
   opacity: 0.0 !important;
 }
 
+body.dark-mode {
+  background: black;
```

Comment:
Might remove this to allow OBS standard background to take effect, or any other client CSS changes